.eslintignore

```diff
@@ -2,4 +2,5 @@ node_modules
 public
 docs
 lib
-build
\ No newline at end of file
+build
```

Comment:
Why did you move this file and `.eslintrc.json` to the root? 🤔

Reply:
because `.eslintrc.json` wasn't picked up by eslint and we couldn't enforce code and formatting consistency across the packages.